pkg/login/social/generic_oauth_test.go

```diff
@@ -427,3 +427,68 @@ func TestUserInfoSearchesForLogin(t *testing.T) {
 		}
 	})
 }
+
+func TestPayloadCompression(t *testing.T) {
+	t.Run("Given a generic OAuth provider", func(t *testing.T) {
```

Comment:
Thank you for keeping the style of the surrounding tests. We're moving over to using testify from having used GoConvey and these tests are somewhere in between 😅

You can remove the outermost layer of subtests if you'd like, but there's no harm done in keeping it.

```suggestion
```